typing/env.ml

```diff
@@ -3277,22 +3283,17 @@ let report_lookup_error _loc env ppf = function
       fprintf ppf
         "The module %a is an alias for module %a, which %s"
         !print_longident lid !print_path p cause
-  | Local_value_escapes(lid, reason) ->
-      let mode =
-        match reason with
-        | `Regionality -> ""
-        | `Locality -> "local "
-      in
-      fprintf ppf
-        "@[The %svalue %a cannot be used here@ \
-           as it escapes its region@]"
-        mode !print_longident lid
-  | Local_value_used_in_closure lid ->
+  | Local_value_used_in_closure (lid, context) ->
       fprintf ppf
         "@[The value %a is local, so cannot be used \
            inside a closure that might escape@]"
-        !print_longident lid
-
+        !print_longident lid;
+      begin match context with
+      | Some Tailcall_argument ->
+         fprintf ppf "@.@[Hint: the closure might escape because it \
```

Comment:
The `the` should be capitalized to match other hints in the compiler.

Reply:
Thanks, fixed.